src/components/fields/NumberField.js

```diff
@@ -60,16 +60,19 @@ class NumberField extends AskField {
 
     if (isCompleted && this.props.validateAs) {
       switch (this.props.validateAs) {
-        case 'email':
-          var emailRegex = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))?$/
-          isValid = emailRegex.test(this.state.value)
-          break
         case 'number':
           isValid = !isNaN(parseFloat(this.state.value)) && isFinite(this.state.value)
           break
       }
     }
 
+    if (this.props.minValue || this.props.maxValue) {
```

Comment:
It should enter the condition if any of those are above zero. I'm not validating negatives here.